docs/source/reference/auto-stop.rst

```diff
@@ -100,6 +104,8 @@ To view the status of the cluster, use ``sky dashboard`` or ``sky status``:
 
 Clusters that are autostopped/autodowned are automatically removed from the status table.
 
+.. _auto-stop-setting-idleness-behavior:
+
 Setting idleness behavior
```

Comment:
We can change the wording a bit for this section.

```
A cluster is considered idle if there is no in-progress jobs (pending or running), or any active SSH connection. You can change the idleness criteria in SkyPilot YAML with :ref:`resources.autostop.wait_for <yaml-spec-resources-autostop>`.

``wait_for`` can be set as one of the followings:
* ``jobs_and_ssh`` (default): wait for in-progress jobs and SSH connections to finish
* ``jobs``: only waits for in-progress jobs - useful for ignoring long-running SSH or IDE connection
* ``none``: waits for nothing, autostop the cluster right after ``idle_minutes`` - useful for ignoring long running jobs, e.g., jupyter notebook, and safeguard the runtime.

Reply:
Oh, just realized we have these breakdown in the reference page as well. It would be nice to mention the useful cases for those items.

Also, it would be fine to repeat these three items here, so people don't need to jump to that reference page to find them.

Reply:
Cool, this looks better! I've also updated the SDK and CLI to be consistent with the new description for each item.

But I purposefully only kept the useful cases for the items in this `auto-stop` page alone.